windows_containers/wmco_rn/windows-containers-release-notes-10-16-x.adoc

```diff
@@ -12,14 +12,13 @@ The release notes for Red{nbsp}Hat OpenShift for Windows Containers tracks the d
 [id="about-windows-containers-numbering"]
 == Windows Machine Config Operator numbering
 
-Y-stream releases of the WMCO are in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.16. Thus, the numbering is WMCO 10.15.z.
+Y-stream releases of the WMCO are in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.16. Thus, the numbering is WMCO 10.16.z.
 
-[id="wmco-10-16-1"]
-== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.16.1
+include::modules/windows-containers-release-notes-10-16-2.adoc[leveloffset=+1]
 
-This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.16.1 were released in link:https://access.redhat.com/errata/RHSA-2024:5749[RHSA-2024:5749].
-
-[id="wmco-10-16-1-bug-fixes"]
-=== Bug fixes
-
-* Previously, if a Windows VM had its PowerShell `ExecutionPolicy` set to `Restricted`, the Windows Instance Config Daemon (WICD) could not run the commands on that VM that are necessary for creating Windows nodes. With this fix, the WICD now bypasses the execution policy on the VM when running PowerShell commands. As a result, the WICD can create Windows nodes on the VM as expected. (link:https://issues.redhat.com/browse/OCPBUGS-37609[*OCPBUGS-37609*])
```

Comment:
Moved this text to [windows-containers-release-notes-10-16-1-0.adoc](https://github.com/openshift/openshift-docs/pull/93954/files#diff-26cf2163e26a24735f11b90822ac8e2c51c30ddd2ca983b3fbb8c8ae079179a1)